recipes/libxml2/all/conanfile.py

```diff
@@ -254,11 +254,10 @@ def package_info(self):
             bindir = os.path.join(self.package_folder, "bin")
             self.output.info("Appending PATH environment variable: {}".format(bindir))
             self.env_info.PATH.append(bindir)
-
-        if self.settings.os in ["Linux", "Macos", "FreeBSD"]:
-            self.cpp_info.system_libs.append('m')
         if self.settings.os == "Windows":
             self.cpp_info.system_libs.append('ws2_32')
+        else:
```

Comment:
I prefer adding Android to the list. there are other systems (e.g. WindowsCE, WindowsPhone, etc.) which might not have libm, we don't want to break them

Reply:
You are right. Done.